pkgs/development/libraries/boxfort/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, gettext, libcsptr, dyncall, nanomsg}:
+
```

Comment:
I would prefer `pkg-config` over `pkgconfig`, as the latter is only an alias of the former, though that might be considered cosmetics...